Examples/MAX78002/CameraIF/main.c

```diff
@@ -210,9 +216,13 @@ int main(void)
         return -1;
     }
 
-    // Start capturing a first camera image frame.
-    printf("Capture image\n");
+    // Start capturing a first  camera image frame.
     camera_start_capture_image();
+    printf("Capture image\n");
+
+#if defined(OV5640_DVP)
+    camera_write_reg(0x503d, 0x0); // workaround: disable test pattern: color bar
```

Comment:
@vicloginov does this still work if you call it _before_ `camera_start_capture_image();`?

Reply:
Modified